src/aeso_fcode_to_fate.erl

```diff
@@ -231,7 +231,30 @@ lookup_var(#env{vars = Vars}, X) ->
 
 %% -- The compiler --
 
+term_to_fate({lit, L}) ->
+     case L of
+        {int, N}             -> aeb_fate_data:make_integer(N);
+        {string, S}          -> aeb_fate_data:make_string(S);
+        {bool, B}            -> aeb_fate_data:make_boolean(B);
+        {account_pubkey, K}  -> aeb_fate_data:make_address(K);
+        {contract_pubkey, K} -> aeb_fate_data:make_contract(K);
+        {oracle_pubkey, K}   -> aeb_fate_data:make_oracle(K);
+        {oracle_query_id, _} -> ?TODO(fate_oracle_query_id_value)
+     end;
+%% negative literals are parsed as 0 - N
+term_to_fate({op, '-', [{lit, {int, 0}}, {lit, {int, N}}]}) ->
+    aeb_fate_data:make_integer(-N);
+term_to_fate(nil) ->
+    aeb_fate_data:make_list([]);
+term_to_fate({op, '::', [Hd | Tl]}) ->
+    aeb_fate_data:make_list([term_to_fate(Hd) | [ term_to_fate(E) || E<-Tl]]);
```

Comment:
```suggestion
    aeb_fate_data:make_list([term_to_fate(Hd) | term_to_fate(Tl)]);
```

Reply:
Works, since `make_list` is identity. Should really do `?FATE_LIST_VALUE` on the tail.